### ISSUE TITLE
[issue]: autosel doesn't work since 1.0.77

### ISSUE THREAD

**Issue by kidmetal** [ventoy/Ventoy#1716]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.78

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### BIOS Mode

UEFI Mode

### Partition Style

MBR

### Disk Capacity

256 GB

### Disk Manufacturer

ScanDisk

### What happened?

autosel doesn't work since 1.0.77.

```
"auto_install": [
        {
            "image": "/Win 10 21H2/Win10.iso",
            "template": "/script/AutoUnattend.xml",
			"autosel": 1
        },
        {
            "image": "/Win 11 22H2/Win11.iso",
            "template": "/script/AutoUnattend_Win11.xml",
			"autosel": 1
        }
    ]
}
```
Ventoy will boot without an installation template. When the version is downgraded to 1.0.76, autosel works perfectly.

Removing autosel and manually selecting the template also works.

**Comment by lzb-libra** [comment]
Does Ventoy support the installation of macos? If not, when will I be able to support it?

**Comment by kidmetal** [comment]
> @kidmetal Please try this CI release: https://github.com/ventoy/Ventoy/actions/runs/2664600194 https://www.ventoy.net/en/doc_github_ci.html

This release works, thanks!

**Comment by ventoy** [comment]
Fixed in the latest 1.0.79 release.
